### DISCUSSION TITLE
I would like to start translating Quarto documentation into Spanish

### DISCUSSION THREAD

**Discussion by statick88** [quarto-dev/quarto-cli#7223] [Feature Requests]
### Description

I would like to start translating Quarto documentation into Spanish, it is a tool that I have been using for about 1 year and 6 months, and I would like to contribute in this way.

There is a lot of Latin American and Spanish speaking community that would benefit a lot with the translation of the documentation to Spanish.

**Comment by mcanouil** [comment]
Thanks for the proposal.

Unfortunately, since Quarto does not support at this point multilingual websites, there is no place to host translations of the whole documentation.
Also, if you were talking about Quarto CLI, same would apply as for now it is not localised.
Additionally this would also require to stay up to date with the main documentation which might be tricky.

In the end, it is not currently really possible right now.
